lib/schema/auth.json

```diff
@@ -463,101 +820,265 @@
         },
         "oauth2": {
             "type": "array",
+            "title": "OAuth2",
+            "description": "Helper attributes for [OAuth2](https://oauth.net/2/)",
             "items": {
                 "oneOf": [
                     {
                         "type": "object",
+                        "description": "The access token for the current request is stored in this field.",
                         "properties": {
                             "key": {
                                 "type": "string",
-                                "enum": ["addTokenTo"]
+                                "enum": ["accessToken"]
                             },
                             "value": {
                                 "type": "string"
+                            },
+                            "type": {
+                                "type": "string",
+                                "enum": ["string"]
+                            }
+                        }
+                    },
+                    {
+                        "type": "object",
+                        "description": "The specifier for token handling behaviour is stored in this field.",
+                        "properties": {
+                            "key": {
+                                "type": "string",
+                                "enum": ["addTokenTo"]
+                            },
+                            "value": {
+                                "type": "string",
+                                "enum": ["header", "queryParams"]
+                            },
+                            "type": {
+                                "type": "string",
+                                "enum": ["string"]
                             }
                         }
                     },
                     {
                         "type": "object",
+                        "description": "This field contains the URL that has to be hit when the authentication process has completed.",
                         "properties": {
                             "key": {
                                 "type": "string",
                                 "enum": ["callBackUrl"]
```

Comment:
callbackUrl ?

Reply:
Yes, this is one of many quirks: https://github.com/postmanlabs/postman-collection-transformer/blob/develop/lib/util.js#L289